main/src/modules/attentionRelated/ears/include/ears.h

```diff
@@ -11,11 +11,14 @@ class ears : public RFModule {
     Port        rpc;
     Port        portToBehavior;
     Port        portToSpeechRecognizer;
+    BufferedPort<Bottle>        port_planner;
+    BufferedPort<Bottle>        goal_need_port;
     BufferedPort<Bottle>        portTarget;
-
+    bool followPlans;
     Mutex m;
     bool bShouldListen;
 
+    std::string      grammarToString(std::string sPath);
```

Comment:
This is included in the `wrdac/functions.h`. No need to re-implement it here! Thus, please remove.